azurepipelines/build/templates/adu-native-build-steps.yml

```diff
@@ -2,99 +2,99 @@
 # Consume this steps template in one or more jobs by passing in parameter values.
 
 parameters:
-  - name: targetOs # example: ubuntu1804
-    type: string
-  - name: targetArch # example: amd64
-    type: string
+    - name: targetOs # example: ubuntu1804
+      type: string
+    - name: targetArch # example: amd64
+      type: string
 
 steps:
-  - ${{ if eq(parameters.targetArch, 'amd64') }}:
-    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
-      displayName: "Component Detection"
-      inputs:
-        sourceScanPath: src
+    - ${{ if eq(parameters.targetArch, 'amd64') }}:
+          - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+            displayName: "Component Detection"
+            inputs:
+                sourceScanPath: src
 
-  - task: Bash@3
-    displayName: "Install ADUC dependencies"
-    inputs:
-      targetType: "filePath"
-      filePath: $(Build.SourcesDirectory)/scripts/install-deps.sh
-      arguments: --install-all-deps
+    - task: Bash@3
+      displayName: "Install ADUC dependencies"
+      inputs:
+          targetType: "filePath"
+          filePath: $(Build.SourcesDirectory)/scripts/install-deps.sh
+          arguments: --install-all-deps
 
-  - task: Bash@3
-    displayName: "Build Client, Unit Tests and Packages: linux-MinSizeRel"
-    inputs:
-      targetType: "filePath"
-      filePath: $(Build.SourcesDirectory)/scripts/build.sh
-      arguments: "--clean --type MinSizeRel --platform-layer linux --log-lib zlog --build-packages --build-unit-tests --static-analysis clang-tidy,cppcheck --out-dir $(Build.BinariesDirectory)"
```

Comment:
Should not need this now since I fixed the clang-tidy issue with--it would be better to keep static analysis on IMO as it does catch real issues: https://github.com/Azure/iot-hub-device-update/pull/313